.golangci.yml

```diff
@@ -4,13 +4,15 @@ run:
 #   timeout: 5m
 
 linters:
-  disable-all: true
+  disable:
```

Comment:
You can't have both `disable` and `enable`, `golangci-lint` will complain sooner or later.

Reply:
It doesn't complain, moreover in the [docs](https://golangci-lint.run/usage/configuration/) it is written differently:
> Slice options (e.g. list of enabled/disabled linters) are combined from the command-line and config file.